.circleci/config.yml

```diff
@@ -9,6 +9,7 @@ jobs:
           name: setup
           command: |
             rustup component add rustfmt --toolchain 1.37.0-x86_64-unknown-linux-gnu
```

Comment:
I think this can be one command `component add rustfmt clippy --toolchain 1.37.0-x86_64-unknown-linux-gnu`